docs/GettingStarted/configuration/ExporterCommittime.md

```diff
@@ -367,4 +386,52 @@ $ oc label image "sha256:${IMAGE_SHA}" "app.kubernetes.io/name=${NAME}"
 
 $ oc annotate image "sha256:${IMAGE_SHA}" --overwrite \
      io.openshift.build.commit.date="${EPOCH_TIMESTAMP}"
-```
\ No newline at end of file
+```
+
+## Container Image Labels support
+Container Image is another method from which the Commit Time Exporter may gather the commit time information **where values from the Build objects required to gather commit time from the source repository are missing**.
+
+The only requirement is to have appropriate LABELs within the Container Image that was build and it's used for the application deployment.
+
+> **Note:** The requirement to add label to the running application as described in the [Deploy Time Exporter](./ExporterDeploytime.md#) still exists.
+
+Below is sample Container image definition that can be in used by the 3rd party CI to adds such labels to the `quay.io/centos7/httpd-24-centos7` from the current project's `git` folder. We will use pelorus project and LABEL our sample `quay.io/pelorus/httpd-sample-app:latest` Container Image with the latest commit hash and commit date:
+
+```Dockerfile
+FROM quay.io/centos7/httpd-24-centos7
+
+ARG LAST_COMMIT_DATE_TIME
+ARG LAST_COMMIT_SHA
+
+LABEL io.openshift.build.commit.date=${LAST_COMMIT_DATE_TIME}
+LABEL io.openshift.build.commit.id=${LAST_COMMIT_SHA}
+LABEL io.openshift.build.source-location="https://github.com/sclorg/httpd-ex"
+```
+
+To build above example, use any container build method, in this example we will use `podman`.
+
+```shell
+git clone https://github.com/dora-metrics/pelorus
```

Comment:
I think is too much to clone repo because of one file, I would simply tell user to create file with the following content and run the following command

Reply:
It is not possible, because in this example we take the last commit hash and last commit time from the git repo to create appropriate LABEL. This is to show how that workflow can be used in real life from any git repository that is building the container image.